README.md

```diff
@@ -1,44 +1,15 @@
-[![Open in Codespaces](https://classroom.github.com/assets/launch-codespace-7f7980b617ed060a017424585567c406b6ee15c891e84e1186181d67ecf80aa0.svg)](https://classroom.github.com/open-in-codespaces?assignment_repo_id=11521484)
-# Sphinx Profile Page
+# Profile Website
 
-This is is a template for a resume/ personal portfolio, built with Sphinx! Based on [Chris Holdgraf's personal site](https://github.com/choldgraf/choldgraf.github.io)
+Welcome to my profile webiste, One Piece Fans
+<!-- enter your target audience after the comma above -->
 
-This template is designed for use in the URI CS TD SSP. 
+Since you're here, you might be: * A employer might go to my website to look at my resume
+* my friend might go on the website to look at the penguin
+* My parents might want to see what I'm up too
+<!-- make a bulleted list of 3 fictional visitors to your site. Include a few detials about them that could impact how you design for them. For each visitor, assign a task or goal they have for visiting your profile website -->
 
+## Design
+I plan on getting my message across by showcasing my resume next to pictures of my project to show what I have done. I also want to add pictures of the anime I like and information about my personality.
 
```

Comment:
For your ## Accessibility section, describe how your site will be accessible to users with different impairments? What web accessibility testing will you run to ensure this?